containers/conformance.md

```diff
@@ -0,0 +1,227 @@
+# Zowe Conformance Criteria Related to Containerization
+
+**DRAFT**
+
+These conformance criteria are applicable for all Zowe components intending to run in a containerized environment. The containerized environment could be Kubernetes or OpenShift running on Linux or Linux on Z. This may also apply to `docker-compose` running on Linux, Windows, Mac OS, or zCX.
+
+## Image
+
+In general, the image should follow [Best practices for writing Dockerfiles](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/). Below requirements are in addition to the list.
```

Comment:
The image SHOULD follow, or MUST follow?

Reply:
Will keep it as SHOULD here. I assume even if they don't follow those best practices, it won't affect us to get certification.

Reply:
As of now it won't affect docker certification because they don't really have requirements. I suppose it could affect us in the future, however.

Reply:
If that's the case, we may need to import the rules here explicitly just in case Docker modify the best practices page. Will check the potential overlaps.